run-build.sh

```diff
@@ -18,14 +18,13 @@ if [[ ! -d $NETLIFY_REPO_DIR ]]; then
 fi
 cd $NETLIFY_REPO_DIR
 
-: ${NODE_VERSION="16"}
-: ${RUBY_VERSION="2.7.2"}
-: ${YARN_VERSION="1.22.10"}
-: ${GO_VERSION="1.16.5"}
-: ${PYTHON_VERSION="3.8"}
```

Comment:
Go and Python defaults have no pinned defaults coming from `buildbot`.  There's little value keeping them here and it only makes things more confusing. Moved it to `run_build_functions` together with `PHP` and `Swift`